tests/foreman/installer/test_infoblox.py

```diff
@@ -4,7 +4,9 @@
 
 :CaseLevel: System
 
-:CaseComponent: DHCPDNS
+:CaseAutomation: Automated
+
+:CaseComponent: Infobloxintegration
```

Comment:
Could you add this to testimony.yml and as well as in CI component.yml?
```suggestion
:CaseComponent: InfobloxIntegration
```

Reply:
Updated

Reply:
As discussed, moving back to DHCPDNS